selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/SeleniumWebDriverHelper.java

```diff
@@ -213,9 +342,30 @@ public WebElement waitPresence(By elementLocator, int timeout) {
    * @return found element
    */
   public WebElement waitPresence(By elementLocator) {
-    return webDriverWaitFactory
-        .get(DEFAULT_TIMEOUT)
-        .until(presenceOfElementLocated(elementLocator));
+    return waitPresence(elementLocator, DEFAULT_TIMEOUT);
+  }
+
+  /**
+   * Waits until all {@link WebElement} with provided {@code elementsLocator} be attached to DOM it
```

Comment:
`be attached` > `are attached`
`it does not mean that elements are visible` > `They are not necessary have to be visible.`